src/overlays/actors/ovl_En_Wf/z_en_wf.c

```diff
@@ -137,81 +144,1344 @@ static ColliderCylinderInit D_80B37A5C = {
+    static Vec3f colliderVec = { 1200.0f, 0.0f, 0.0f };
+    static Vec3f bodyPartVec = { 0.0f, 0.0f, 0.0f };
+    EnWf* this = THIS;
+    s32 bodyPartIndex = -1;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B35B94.s")
+    Collider_UpdateSpheres(limbIndex, &this->colliderSphere);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B35C10.s")
+    if (limbIndex == 6) {
+        Vec3f colliderPos;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B35D18.s")
+        bodyPartIndex = -1;
+        Matrix_MultVec3f(&colliderVec, &colliderPos);
+        this->colliderCylinder2.dim.pos.x = colliderPos.x;
+        this->colliderCylinder2.dim.pos.y = colliderPos.y;
+        this->colliderCylinder2.dim.pos.z = colliderPos.z;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B35D90.s")
+    if ((this->fireTimer != 0) || ((this->actor.colorFilterTimer != 0) && (this->actor.colorFilterParams & 0x4000))) {
+        switch (limbIndex) {
+            case WOLFOS_LIMB_EYES:
+                bodyPartIndex = 0;
+                break;
+            case WOLFOS_LIMB_FRONT_RIGHT_LOWER_LEG:
+                bodyPartIndex = 1;
+                break;
+            case WOLFOS_LIMB_FRONT_LEFT_LOWER_LEG:
+                bodyPartIndex = 2;
+                break;
+            case WOLFOS_LIMB_THORAX:
+                bodyPartIndex = 3;
+                break;
+            case WOLFOS_LIMB_ABDOMEN:
+                bodyPartIndex = 4;
+                break;
+            case WOLFOS_LIMB_TAIL:
+                bodyPartIndex = 5;
+                break;
+            case WOLFOS_LIMB_BACK_RIGHT_SHIN:
+                bodyPartIndex = 6;
+                break;
+            case 37: // There is no limb with index this large
+                bodyPartIndex = 7;
+                break;
+            case WOLFOS_LIMB_BACK_RIGHT_PASTERN:
+                bodyPartIndex = 8;
+                break;
+            case WOLFOS_LIMB_BACK_LEFT_PAW:
+                bodyPartIndex = 9;
+        }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B35E4C.s")
+        if (bodyPartIndex >= 0) {
+            Vec3f bodyPartPos;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B35EE4.s")
+            Matrix_MultVec3f(&bodyPartVec, &bodyPartPos);
+            this->bodyPartsPos[bodyPartIndex].x = bodyPartPos.x;
+            this->bodyPartsPos[bodyPartIndex].y = bodyPartPos.y;
+            this->bodyPartsPos[bodyPartIndex].z = bodyPartPos.z;
+        }
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Wf/func_80B360E8.s")
+static void* sWolfosEyeTextures[] = { gWolfosNormalEyeOpenTex, gWolfosNormalEyeHalfTex, gWolfosNormalEyeNarrowTex,
+                                      gWolfosNormalEyeHalfTex };
+static void* sWhiteWolfosEyeTextures[] = { gWolfosWhiteEyeOpenTex, gWolfosWhiteEyeHalfTex, gWolfosWhiteEyeNarrowTex,
```

Comment:
```suggestion
static void* sWolfosNormalEyeTextures[] = { gWolfosNormalEyeOpenTex, gWolfosNormalEyeHalfTex, gWolfosNormalEyeNarrowTex,
                                      gWolfosNormalEyeHalfTex };
static void* sWolfosWhiteEyeTextures[] = { gWolfosWhiteEyeOpenTex, gWolfosWhiteEyeHalfTex, gWolfosWhiteEyeNarrowTex,
```
for consistency with asset symbols